iati/default.py

```diff
@@ -88,6 +90,8 @@ def codelist(name, version=None):
 
         Better distinguish TypeErrors from KeyErrors - sometimes the latter is raised when the former should have been.
 
+        Determine how to handle requested an Embedded Codelist for an Integer Version.
```

Comment:
`how to handle requested an Embedded Codelist` - doesn't seem to make sense?

Reply:
This relates to:

* Embedded Codelists relate to a Decimal
* Integers are not a decimal
* Should the function distinguish between embedded and NE codelists?

The answer to this is: treat an integer value like the most recent decimal within it.